### DISCUSSION TITLE
Upgrade Module

### DISCUSSION THREAD

**Discussion by Vandal593** [OCA/OpenUpgrade#3814] [Announcements]
Hi, I am upgrading Odoo 10 to Odoo11, but I have a question about the custom modules, how do I upgrade these modules, and is the upgrade only to the database or can I upgrade code as well?

**Comment by pedrobaeza** [comment]
Wrong discussion board. Please resubmit as discussion and in English.

**Reply by Vandal593** [reply]
Ok, sorry
